ft-email-auth-provider/src/handlers/create_account.rs

```diff
@@ -7,6 +7,7 @@ struct CreateAccount {
     email_confirmation_code: String,
     user_id: Option<ft_sdk::UserId>,
     email_confirmation_sent_at: chrono::DateTime<chrono::Utc>,
+    pre_verified: bool,
```

Comment:
Write a documentation of this field. It's not so clear what it means.

Reply:
done in https://github.com/fastn-community/auth/pull/7/commits/80dbb8ceccaede62c4e2757edbe7a0e545f3e2bc